patch/llpcPatchEntryPointMutate.cpp

```diff
@@ -408,14 +408,27 @@ FunctionType* PatchEntryPointMutate::GenerateEntryPointType(
              // and indirect user data should not be counted in possible spilled user data.
             if (pNode->type == ResourceMappingNodeType::IndirectUserDataVaPtr)
             {
-                // Only the first shader stage needs a vertex buffer table.
-                if ((m_pContext->GetShaderStageMask() & (ShaderStageToMask(m_shaderStage) - 1)) == 0)
+                // Only the vertex shader needs a vertex buffer table.
```

Comment:
This commit shouldn't be included in this pull request, right? I see you have made a separate one.

Reply:
Correct, but I have to include both commits in the pull request for the testing to work. Once pr89 has been pulled, I can rebase this pr so only one commit remains.

Reply:
Understand. You can make comments in the pull request :)

Reply:
(I mean pr88.)